app/models/movie.rb

```diff
@@ -10,22 +11,16 @@ class Movie < ActiveRecord::Base
   has_many :listings
   has_many :lists, through: :listings
   has_many :users, through: :listings
-
-  def self.by_tag_and_user(tag, user)
```

Comment:
The changes in this model are just to rearrange the code a little bit to put like-things near each other.